test-apps/imodel-transformer/src/Transformer.ts

```diff
@@ -66,6 +66,29 @@ export class Transformer extends IModelTransformer {
     transformer.logElapsedTime();
   }
 
+  /** attempt to isolate a set of elements by transforming only them from the source to the target */
```

Comment:
this PR was based off the isolated elements PR to the transformer test-app, will rebase out those changes